### PR TITLE
feat: making dynamic tile size calculation per tile format: jpeg/png (MAPCO-2758)

### PR DESCRIPTION
**Pull request by ronenkapelian**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

* Removing Hard-Coded option of calculate based on jpeg estimated size 

* Making dynamic cacl according layer's tiles output format

* Now should provide configuration of PNG + JPEG single tile size

### CODE REVIEW

File: src/createPackage/models/createPackageManager.ts [hunk 1]
```diff
@@ -371,4 +370,16 @@ export class CreatePackageManager {
 
     return newPolygonLarts;
   }
+
+  private getTileEstimatedSize(tileOutputFormat: TileOutputFormat): number {
+    let tileEstimatedSize;
+    if (tileOutputFormat === TileOutputFormat.JPEG) {
+      tileEstimatedSize = this.config.get<number>('storageEstimation.jpegTileEstimatedSizeInBytes');
```
Comment (AnguIar) [inline src/createPackage/models/createPackageManager.ts]: interface bitch

**Review by AnguIar — CHANGES_REQUESTED** [review]

File: src/createPackage/models/createPackageManager.ts (hunk 1 above)
Comment (ronenkapelian) [reply src/createPackage/models/createPackageManager.ts]: done b-yatch

**Review by AnguIar — APPROVED** [review]
WHY?

**Comment by AnguIar** [comment]
:100: :1st_place_medal:

**Pull request merged**